ui/lib/core/addon/helpers/engine-options-for-backend.js

```diff
@@ -0,0 +1,36 @@
+import { helper as buildHelper } from '@ember/component/helper';
+import { capitalize } from '@ember/string';
+import { assign } from '@ember/polyfills';
+
+const SECRET_BACKENDS = {
+  pki: {
+    displayName: 'PKI',
+    navigateTree: false,
+    tabs: [
+      {
+        label: 'Overview',
+        link: 'overview',
+      },
+    ],
+  },
+};
+
+export function engineOptionsForBackend([backend, tab]) {
+  const selected = SECRET_BACKENDS[backend];
+  let backendOptions;
+
+  if (selected && selected.tabs) {
+    let tabData =
+      selected.tabs.findBy('name', tab) || selected.tabs.findBy('modelPrefix', tab) || selected.tabs[0];
+    backendOptions = assign({}, selected, tabData);
+  } else if (selected) {
+    backendOptions = selected;
+  } else {
+    backendOptions = assign({
+      displayName: capitalize(backend),
+    });
+  }
+  return backendOptions;
+}
+
+export default buildHelper(engineOptionsForBackend);
```

Comment:
It looks like this is pretty much the same as `options-for-backend` but the `pki` data is different. Is the primary issue that there is a naming collision with the current pki? I'm wondering if we could continue to use `option-for-backend` and add this pki data under a different key like `enginePki` for now?

In `SecretListHeader`, `@model.engineType` was originally passed into `options-for-backend`. Could you change that to a getter that checks for `isEngine` and then return either `enginePki` or `@model.engineType`? I think that would simplify this and save from duplicating this helper.